### PR TITLE
Switch # artifacts to millions

### PR DESCRIPTION
**Pull request by jaimergp**
PR Checklist:

- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] put any other relevant information below

conda-forge is now over 1M artifacts! That means conda-forge.org renders it as 1011k artifacts... and 1M looks cooler :D

### CODE REVIEW

File: js/stats.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ function displayStatsJSON (reportText) {
   div.innerHTML = (
     (stats.n_repos / 1e3).toFixed(1) + 'k feedstocks<br>' +
     (stats.n_packages / 1e3).toFixed(1) + 'k packages<br>' +
```
Comment (jakirkham) [inline js/stats.js:36]: Should we update these too?

File: js/stats.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ function displayStatsJSON (reportText) {
...
-    (stats.n_artifacts / 1e3).toFixed(0) + 'k artifacts'
+    (stats.n_artifacts / 1e6).toFixed(1) + 'M artifacts'
```
Comment (jakirkham) [inline js/stats.js:37]: Is there a way to automate bumping this? Perhaps some JavaScript package that renders numbers in scientific format or similar?

**Review by jakirkham — COMMENTED** [review]
Thanks Jaime! 🙏

Had a couple follow up questions below

File: js/stats.js (hunk 2 above)
Comment (jaimergp) [reply js/stats.js:37]: This is how it looks like now:

image


I don't anticipate another change anytime soon, but I guess we could have a little function to automate the formatting.

Comment (jaimergp) [reply js/stats.js:37]: e.g. like this https://stackoverflow.com/a/60550008/3407590

**Pull request merged**